mirrord/agent/src/file.rs

```diff
@@ -472,6 +485,67 @@ impl FileManager {
             })
     }
 
+    pub(crate) fn mkdir(&mut self, path: &Path, mode: u32) -> RemoteResult<MakeDirResponse> {
```

Comment:
Instead of using separate calls to `std::fs::create_dir` and `fs::set_permissions` in this function, you can use one call to [`nix::unistd::mkdir`](https://docs.rs/nix/latest/nix/unistd/fn.mkdir.html) (requires enabling `fs` feature on `nix` dependency)

Reply:
👍 Done, thanks